CellProfiler_pipelines/Pipelines/NF1_analysis_cp_plugin.cppipe

```diff
@@ -88,7 +88,7 @@ RunCellpose:[module_num:5|svn_version:'Unknown'|variable_revision_number:3|show_
     Invert images:No
     Remove objects that are touching the edge?:Yes
 
-RunCellpose:[module_num:6|svn_version:'Unknown'|variable_revision_number:3|show_window:False|notes:[]|batch_state:array([], dtype=uint8)|enabled:True|wants_pause:False]
+RunCellpose:[module_num:6|svn_version:'Unknown'|variable_revision_number:3|show_window:False|notes:['Same as previous module.']|batch_state:array([], dtype=uint8)|enabled:True|wants_pause:False]
```

Comment:
I think it would be clearer to copy the description from the last file here in addition to the (same as previous module).

Reply:
Agreed, I have changed it and added the why to turning it too (e.g. thought it should be) in commit [b1f5958](https://github.com/WayScience/NF1_SchwannCell_data/pull/42/commits/b1f59587b6e32321559f1514c6710f0363369155)